amethyst_gltf/src/format/importer.rs

```diff
@@ -0,0 +1,328 @@
+use std;
+use std::error::Error as StdError;
+use std::fmt;
+use std::path::Path;
+use std::sync::Arc;
+
+use assets::BoxedErr;
+use assets::Source as AssetSource;
+use base64;
+use gltf;
+use gltf::Gltf;
+use gltf::json;
+use gltf::json::validation;
+use gltf_utils::Source;
+
+#[derive(Debug)]
+pub enum ImageFormat {
```

Comment:
Could we add `from_mime` or something like that?

Reply:
Done